validate/src/message.rs

```diff
@@ -16,6 +16,12 @@ use twilight_model::{
     id::{marker::StickerMarker, Id},
 };
 
+/// ASCII dash.
+pub const DASH: char = '-';
+
+/// ASCII dot.
+pub const DOT: char = '.';
+
```

Comment:
I am not sure we need these to be public (also including `UNDERSCORE`) could probably be inlined? or do you have any specific thoughts with making them public?

Reply:
I didn't really know for sure whether or not, but I suppose they are pretty much useless, so I will make them private.